Fix/backprojection

jlegrand62 opened the pull request:
Resolve #58, resolve #35

jlegrand62 committed:
Update

jlegrand62 committed:
Try to debug (add sleep time to get debug message?)

jlegrand62 committed:
Stil no debug message...

jlegrand62 committed:
Fix init_buffers for backprojection

Buffer array initialized with correct dtype!

jlegrand62 committed:
Removed unused debug messages

jlegrand62 committed:
Update vol2pcd to get progress bar

jlegrand62 committed:
Fix debug message

naittaleb reviewed (changes requested):
Print allocated memory to warn user about possible memory overflow

hanappe commented on romiscan/cl.py:
```diff
@@ -75,24 +75,25 @@ def __init__(self, shape, origin, voxel_size, type="carving", default_value=0, l
         """
         Helper function to initialize OpenCL buffers.
         """
         self.values_h = self.default_value * \
```
Print out the sizes of self.shape and the total memory requiements (WxHxPx4 bytes).

hanappe reviewed (changes requested):
Please add a print with the request shape in init_buffer + the total required memory size.

jlegrand62 committed:
Add logger info on buffer array size in brackprojection

jlegrand62 committed:
Add buffer array shape

jlegrand62 committed:
Proper Python code for Backprojection class

If you don't agree, check the PEP guidelines & the Numpy docstring style conventions.
